.azure/pipelines/ci.yml

```diff
@@ -176,7 +176,8 @@ stages:
                 -noBuildDeps
                 $(_BuildArgs)
                 $(_InternalRuntimeDownloadArgs)
-        condition: ne(variables['Build.Reason'], 'PullRequest')
+        # Disabled until 3.1.3 is released
+        condition: false
```

Comment:
Suggest this can't be merged except on its own. There's no validation of the remainder of your change.

Reply:
The remainder of the change can't be validated because the site extension wont fully build until the next 3.1 patch is released.

Reply:
`<swear>favourite words</swear>`